tests/unit/cluster/cluster-migrateslots.tcl

```diff
@@ -1066,9 +1110,13 @@ start_cluster 3 3 {tags {logreqres:skip external:skip cluster} overrides {cluste
         assert_match "500" [R 2 CLUSTER COUNTKEYSINSLOT 16383]
         assert_match "500" [R 5 CLUSTER COUNTKEYSINSLOT 16383]
 
-        # Expect error messages
-        assert {[string match {*I was demoted to a replica*} [dict get [get_migration_by_name 0 $jobname] message]]}
-        assert {[string match {*Connection lost to target*} [dict get [get_migration_by_name 2 $jobname] message]]}
+        # Expect error messages. Either a failover, or if the PSYNC fails, it
+        # will be a full resync message.
```

Comment:
Why will the psync fail? I would rather remove the ways the pysnc to avoid silent regressions here. We had an issue in ElastiCache for a long time where we desynced but the psync succeeded, but was always silent in our tests.